src/components/Thumbnail/Image.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { FC } from "react";
+import classnames from "classnames";
+
+interface IProps {
+  icon: string;
+  alt: string;
+  classname?: string;
+}
+export const Image: FC<IProps> = ({ icon, alt, classname = "" }) => (
```

Comment:
we don't need now default value for classname because classnames library fix this